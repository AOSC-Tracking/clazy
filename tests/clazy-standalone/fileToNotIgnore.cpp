#include <QtCore/QDateTime>

void t()
{
    QDateTime::currentDateTime().toSecsSinceEpoch();
}
