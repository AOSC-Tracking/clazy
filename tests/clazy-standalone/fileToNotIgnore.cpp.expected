clazy-standalone/fileToNotIgnore.cpp:5:5: warning: Use QDateTime::currentSecsSinceEpoch() instead. It is significantly faster [-Wclazy-qdatetime-utc]
